crates/trigger/src/cli.rs

```diff
@@ -24,69 +24,67 @@ pub const SPIN_WORKING_DIR: &str = "SPIN_WORKING_DIR";
 
 /// A command that runs a TriggerExecutor.
 #[derive(Parser, Debug)]
-#[clap(next_help_heading = "TRIGGER OPTIONS")]
+#[command(next_help_heading = "TRIGGER OPTIONS")]
 pub struct TriggerExecutorCommand<Executor: TriggerExecutor>
 where
     Executor::RunConfig: Args,
 {
     /// Log directory for the stdout and stderr of components.
-    #[clap(
-            name = APP_LOG_DIR,
-            short = 'L',
-            long = "log-dir",
-            )]
+    #[arg(
+        short = 'L',
+        long = "log-dir",
+        id = APP_LOG_DIR,
+    )]
     pub log: Option<PathBuf>,
 
     /// Disable Wasmtime cache.
-    #[clap(
-        name = DISABLE_WASMTIME_CACHE,
+    #[arg(
         long = "disable-cache",
+        id = DISABLE_WASMTIME_CACHE,
         env = DISABLE_WASMTIME_CACHE,
-        conflicts_with = WASMTIME_CACHE_FILE,
-        takes_value = false,
+        conflicts_with = WASMTIME_CACHE_FILE
     )]
     pub disable_cache: bool,
 
     /// Wasmtime cache configuration file.
-    #[clap(
-        name = WASMTIME_CACHE_FILE,
+    #[arg(
         long = "cache",
+        id = WASMTIME_CACHE_FILE,
         env = WASMTIME_CACHE_FILE,
         conflicts_with = DISABLE_WASMTIME_CACHE,
     )]
     pub cache: Option<PathBuf>,
 
     /// Print output for given component(s) to stdout/stderr
-    #[clap(
-        name = FOLLOW_LOG_OPT,
+    #[arg(
         long = "follow",
-        multiple_occurrences = true,
-        )]
+        id = FOLLOW_LOG_OPT
+    )]
     pub follow_components: Vec<String>,
 
     /// Print all component output to stdout/stderr
-    #[clap(
+    #[arg(
         long = "follow-all",
-        conflicts_with = FOLLOW_LOG_OPT,
-        )]
+        conflicts_with = FOLLOW_LOG_OPT
+    )]
     pub follow_all_components: bool,
 
     /// Set the static assets of the components in the temporary directory as writable.
-    #[clap(long = "allow-transient-write")]
+    #[arg(long)]
     pub allow_transient_write: bool,
 
     /// Configuration file for config providers and wasmtime config.
-    #[clap(
-        name = RUNTIME_CONFIG_FILE,
+    #[arg(
```

Comment:
Optional: Noticing that we should be able to reduce to `long` in the line below to inherit `"runtime-config-file"` by default, right?

Reply:
The long will take the id and convert _ to - , but won't convert to lowercase. So in this case that won't work since it's all caps behind that const...